### ISSUE TITLE
More info

### ISSUE THREAD

**Issue by Derpmore** [Saegusae/looks#1]
Hi!

Would it be possible to implement information about which face was used and the adornment aswell? Right now the save command only puts in the detailed info. Since different faces give different outcome if you follow the detailed settings.
 
Thanks for a rlly nice mod! :D

**Comment by SaltyMonkey** [comment]
it need so complicated work. Doubt...
Its not how coding works - you cant see "presets" in packet.

**Comment by Owyn** [comment]
I agree, saving face\hair\other presets would be much more useful than just sliders
